typedb-src/modules/ROOT/pages/quickstart-guide.adoc

```diff
@@ -0,0 +1,324 @@
+----
+--
+TypeDB Studio::
++
+--
+Open the xref:attachment$iam-data.tql[iam-data.tql] file from the Project panel.
+
+Execute all Insert queries from the file:
+
+. Ensure the Session toggle is set to `data` and the Transaction toggle is set to `write`.
+. Execute all queries in the file by clicking btn:[Run Query].
+. When the queries have completed, click btn:[Commit Transaction].
+--
+====
+
+Assuming there were no errors in the process, the data is persisted in the database.
+The database is ready for queries now.
+If you want, try some <<_query_examples,query examples>> below with the new database.
+
+[NOTE]
+====
+The IAM schema and data from this Quickstart guide are used throughout the documentation.
+====
+
+[#_query_examples]
+== Query examples
+
+Try sending the following queries:
+
+* <<#_get_query__retrieve_all_data>>
+* <<#_get_query__retrieve_all_schema_types>>
+* <<#_get_query__get_all_emails_of_kevin>>
+* <<#_insert_query__add_a_new_email_for_kevin>>
+* <<#_update_query__change_the_newly_added_email_for_kevin>>
+* <<#_delete_query__delete_ownership_of_the_newly_updated_email_from_kevin>>
+
+//#todo Replace the link below with a more specific one in a TypeQL section. Query types?
+For more TypeQL query examples, check the xref:typeql:ROOT:overview.adoc[TypeQL] documentation.
+//#todo Add the TypeDB in 20 queries page link
+
+[#_get_query__retrieve_all_data]
+=== Get all data
+
+To retrieve all data from a database:
+
+[,typeql]
+----
+match $t isa thing;
+----
+
+[#_get_query__retrieve_all_schema_types]
+=== Get all types
+
+To retrieve all types from a schema of a database:
+
+[,typeql]
+----
+match $t sub thing;
+----
+
+[#_get_query__get_all_emails_of_kevin]
+=== Get all emails of Kevin
+
+To retrieve all emails owned by every person with a full name that contains "Kevin"
+(we have only one such person in the example database):
+
+[,typeql]
+----
+match
+    $p isa person,
+        has full-name $fn;
+    $fn contains "Kevin";
+    $p has email $e;
+get $e;
+----
+
+[#_insert_query__add_a_new_email_for_kevin]
+=== Insert a new email for Kevin
+
+To insert a new email (`kevin@gmail.com`) for every person with a full name that contains "Kevin":
+
+[,typeql]
+----
+match
+    $p isa person,
+        has full-name $fn;
+    $fn contains "Kevin";
+insert $p has email "kevin@gmail.com";
+----
+
+Now Kevin has more than one email.
+
+[#_update_query__change_the_newly_added_email_for_kevin]
+=== Update the email for Kevin
+
+To update the email from `kevin@gmail.com` to `kevin2@gmail.com` for every person with a full name that contains "Kevin":
+
+[,typeql]
+----
+match
+    $p isa person,
+        has full-name $fn,
+        has email $e;
+    $fn contains "Kevin";
+    $e = "kevin@gmail.com";
+delete $p has $e;
+insert $p has email "kevin2@gmail.com";
+----
+
+The query above does the following:
+
+- Matches the person and its email with the value of `kevin@gmail.com`.
```

Comment:
```suggestion
- Matches the person and its email, requiring that the value is `kevin@gmail.com`.
```

Reply:
The value of what? It became even more ambiguous that way, IMHO. Don't you think?

Reply:
```suggestion
- Matches the person and its email (`kevin@gmail.com`).
```